### PR TITLE
fix: static links replaced with clickable inside appointment description

### PR DESCRIPTION
**Pull request by geeky-abhishek**

### CODE REVIEW

File: src/commons/body-message-renderer.jsx [hunk 1]
```diff
@@ -3,22 +3,35 @@
...
-import React, { useCallback, useEffect, useRef } from 'react';
-import { forEach, reduce } from 'lodash';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+import { forEach, reduce, map } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Container, Text } from '@zextras/carbonio-design-system';
 
 const _CI_REGEX = /^<(.*)>$/;
 const _CI_SRC_REGEX = /^cid:(.*)$/;
+const LINK_REGEX = /\bhttps?:\/\/\S+/g;
```
Comment (zovomat) [inline src/commons/body-message-renderer.jsx]: this regex doesn't match urls that don't include the protocol (e.g. `mail.zextras.com/zx/team/user-room/.....`),  is this behavior wanted?

File: src/commons/body-message-renderer.jsx [hunk 2]
```diff
@@ -3,22 +3,35 @@
...
-	useEffect(() => {
-		containerRef.current.innerText = text;
-	}, [text]);
+export const locationUrl = (location) => {
+	const found = location?.match(LINK_REGEX);
+	return found?.length ? found[0] : undefined;
+};
 
-	return <Text ref={containerRef} />;
+function TextMessageRenderer({ text }) {
```
Comment (zovomat) [inline src/commons/body-message-renderer.jsx:20]: in this case only the urls inside the text are replaced with links, iirc the requirement was to do it in the appt location, right?

**Review by zovomat — DISMISSED** [review]

File: src/commons/body-message-renderer.jsx (hunk 1 above)
Comment (geeky-abhishek) [reply src/commons/body-message-renderer.jsx]: I'm not sure but I'll confirm from @AlessioCrestani

Comment (zovomat) [reply src/commons/body-message-renderer.jsx]: unresolving because i haven't seen any discussion or resolution on this

Comment (giuliano176) [reply src/commons/body-message-renderer.jsx]: if you need it here you can find the regex for links validation
https://www.regextester.com/93652

**Review by giuliano176 — CHANGES_REQUESTED** [review]

File: src/commons/body-message-renderer.jsx (hunk 1 above)
Comment (zovomat) [reply src/commons/body-message-renderer.jsx]: pinging @AlessioCrestani

**Comment by geeky-abhishek** [comment]
@zovomat Carlo confirmed that we don't want to handle the urls without protocol (for now)

**Comment by geeky-abhishek** [comment]
Thanks @giuliano176 for this https://www.regextester.com/93652 ,currently I'm making some changes to this approach once done I'll update the correct regex.

**Review by zovomat — DISMISSED** [review]

**Review by giuliano176 — DISMISSED** [review]

**Review by zovomat — APPROVED** [review]

**Review by giuliano176 — APPROVED** [review]

**Review by nubsthead — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

**Pull request merged**